solo-chains/node/tanssi-relay-service/Cargo.toml

```diff
@@ -233,7 +233,3 @@ runtime-metrics = [
 	"polkadot-runtime-parachains/runtime-metrics",
 	"starlight-runtime?/runtime-metrics",
 ]
-
```

Comment:
You removed this because now because elastic scaling is officially integrated (i.e not an experiment right?)

Reply:
Yes, they removed the feature, it was a compile error to have it here.

Reply:
*resolved*